docs/devel/reporting.rst

```diff
@@ -1,14 +1,14 @@
 Translation progress reporting
 ==============================
 
-Reporting features give insight into how a translation progresses over a given
-period. A summary of contributions to any given component over time is
-provided. The reporting tool is found in the :guilabel:`Insights` menu of any
-translation component, project or on the dashboard:
+Reporting features give insight into translation progress over a given period.
+A summary of contributions to any given component over time is provided.
+The reporting tool is found in the :guilabel:`Insights` menu of the dashboard:,
+or any translation component, or project:
```

Comment:
```suggestion
any translation component, or project:
```
Ref. https://github.com/WeblateOrg/weblate/pull/7223/commits/7334d6625d0426e43c18910dece3ce2cfb2ef57b
Otherwise implies it goes missing elsewhere if found in other places.